lib/RoutingList/RoutingList.js

```diff
@@ -0,0 +1,108 @@
+import { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+
+import {
+  Accordion,
+  Badge,
+  Button,
+  Dropdown,
+  DropdownMenu,
+  Loading,
+  MenuSection,
+} from '@folio/stripes/components';
+
+import {
+  columnMapping,
+  CREATE_ROUTING_LIST_URL,
+  ROUTING_LIST_ACCORDION_ID,
+  routingListFormatter,
+  visibleColumns,
+} from './constants';
+import { useRoutingList } from './hooks';
+import { RoutingListTable } from './RoutingListTable';
+
+export const RoutingList = ({ actionMenu, disabled, poLineId }) => {
+  const intl = useIntl();
+
+  const { isLoading, routingLists } = useRoutingList(poLineId);
+
+  const defaultActionMenu = useMemo(() => (
+    <Dropdown
+      data-testid="routing-list-action-dropdown"
```

Comment:
I do not see any usage of `routing-list-action-dropdown` test id. Could you remove it?